scripts/async_profiler_build_shared.sh

```diff
@@ -5,10 +5,10 @@
 #
 set -euo pipefail
 
-VERSION=v2.9g4
-GIT_REV="cde6aead3f8c30ccc79eaa3a44f36ed97b8ebc91"
+VERSION=remove-our-semicolon-patch
```

Comment:
Let's update it to v2.9g5 60cbc128a752c657e8a97300784686e08c8dbe9d

Reply:
👍